document_ai_warehouse_processing_python/document_warehouse_utils.py

```diff
@@ -355,14 +322,7 @@ def update_document_schema(self, schema_id: str, schema: str):
         request.name = f"{parent}/documentSchemas/{schema_id}"
 
         # define schema
-        request.document_schema = contentwarehouse_v1.DocumentSchema()
-        request.document_schema.display_name = schema_json["display_name"]
-        request.document_schema.document_is_folder = schema_json["document_is_folder"]
-        request.document_schema.description = schema_json["description"]
-
-        for property_definition in schema_json["property_definitions"]:
-            request.document_schema.property_definitions.append(
-                self.create_document_schema_property_definition(property_definition))
+        request.document_schema = contentwarehouse_v1.DocumentSchema.from_json(schema_json)
```

Comment:
Glad this worked! Greatly simplified this function.

(This type of conversion isn't obvious in the client libraries)